src/types.ts

```diff
@@ -5,6 +5,8 @@ interface SharedOpts {
   tsconfig?: string;
   // Is error extraction running?
   extractErrors?: boolean;
+  // Is executive?
```

Comment:
```suggestion
  // Is executable?
```
"executive" means something different